robot_arm:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  joint1_position_controller:
    type: position_controllers/JointPositionController
    joint: joint1
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint2_position_controller:
    type: position_controllers/JointPositionController
    joint: joint2
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint3_position_controller:
    type: position_controllers/JointPositionController
    joint: joint3
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint4_position_controller:
    type: position_controllers/JointPositionController
    joint: joint4
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint5_position_controller:
    type: position_controllers/JointPositionController
    joint: joint5
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint6left_position_controller:
    type: position_controllers/JointPositionController
    joint: joint6_left
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint7left_position_controller:
    type: position_controllers/JointPositionController
    joint: joint7_left
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint8left_position_controller:
    type: position_controllers/JointPositionController
    joint: joint8_left
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint9right_position_controller:
    type: position_controllers/JointPositionController
    joint: joint9_right
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint10right_position_controller:
    type: position_controllers/JointPositionController
    joint: joint10_right
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint11right_position_controller:
    type: position_controllers/JointPositionController
    joint: joint11_right
    pid: {p: 100.0, i: 0.01, d: 10.0}
